bookShop/src/app/state/books.actions.ts

```diff
@@ -0,0 +1,14 @@
+import { createAction, props } from '@ngrx/store';
+import { IBook } from '../shared/models/BookModel';
+
+export const addBook = createAction('[Book List] Add Book', props<IBook>());
+
+export const removeBook = createAction(
+  '[Book Collection] Remove Book',
+  props<{ bookId: any }>()
```

Comment:
You need to specify the type of `bookId`

Reply:
Fixed